rpcs3/Emu/RSX/VK/VKTextureCache.cpp

```diff
@@ -574,6 +574,7 @@ namespace vk
 			view_swizzle = source->native_component_map;
 		}
 
+		image->set_debug_name("Temp view");
```

Comment:
are these meant for master ?
just wondering.

Reply:
Yes, they show up in renderdoc so its good to keep them.